README.md

```diff
@@ -29,6 +29,25 @@ docker exec coursework_attacker python attack.py
 To see Flask app: navigate to http://localhost:5000
 
 To see DNS admin panel: navigate to https://localhost:10000/ and log in with `root`/`highentropy`
+## Wireshark
+If Wireshark is not installed, run:
+```bash
+sudo apt install wireshark-qt
+```
+To access Wireshark, cd to where the tcpdump folder is and run:
+```bash
+tail -c +1 -f tcpdump/tcpdump.pcap | wireshark -k -i -
+```
+To see the dns packets for our network, in the filter, type ip.addr == 172.16.238.10
+
+If you get access denied:
+```bash
+sudo dpkg-reconfigure wireshark-common
+```
+Select yes, then:
+```bash
+sudo dpkg-reconfigure wireshark-common
+```
```

Comment:
are the two commands meant to be the same?

Reply:
I changed from https://github.com/ffeldhaus/docker-wireshark to https://hub.docker.com/r/kaazing/tcpdump. I coulnd't make the first one work. It didn't save the traffic for the docker network.

For the commands, you first type the first command, then a box appears which you have to select yes or no. Then you type the second command

Reply:
Ah nice catch, yea sorry the second command should be: sudo chmod +x /usr/bin/dumpcap